contents/teams/experiments/objectives.mdx

```diff
@@ -1,19 +1,36 @@
-### Q4 2024 Objectives
-
-This quarter, we're focusing on more feature improvements to help users create comprehensive and easy-to-analyze experiments. These improvements will make our product more mature and complete.
-
-### Objective 1: Finish HogQL rewrite
-
-We will migrate Experiments to HogQL, making result calculations more reliable and performant. This will also enable the addition of new features listed below.
-
-### Objective 2: Multiple experiment goals supported + visualized
-
-We will add support for multiple goal metrics in a single experiment, allowing them to be visualized together and making it easier to interpret results across all metrics simultaneously.
-
-### Objective 3: Reusable experiment metrics
-
-We will add the ability to create metric sets that users can save and reuse. This will reduce friction, improve maintainability, and decrease the likelihood of errors.
-
-### Objective 4: Review and adjust methodology with a statistician
-
-We will review our current methodology to ensure it is both accurate and easy for our users to understand.
+### Q1 2025 Objectives
+
+This quarter, we will bring new feature improvements to make Experiments more advanced. We'll also focus on improving quality across different areas - data, documentation, and codebase - to help us move faster and deliver a better product.
+
+#### Objective 1: New features
+- [Timeseries chart for deltas and credible intervals](https://github.com/PostHog/posthog/issues/26931) @jurajmajerik 
+- [Winsorization](https://github.com/PostHog/posthog/issues/26060) to exclude extreme values from analyses. @andehen 
+- [New data collection calculator](https://github.com/PostHog/posthog/issues/26933) @andehen 
+- [Filtering for experiment results](https://github.com/PostHog/posthog/issues/26934) @danielbachhuber 
+
+#### Objective 2: Data & statistics
+- Complete the [migration to updated statistical methods](https://github.com/PostHog/posthog/issues/26713).
+- Document each statistical method in detail.
+- Write a practical "manual" chapter for every metric type.
+- Address neglected cases, such as "average count per user."
+- Improve diagnostics to provide more detailed information, like flagging missing exposure data. @andehen 
+
+#### Objective 3: Data Warehouse integration @danielbachhuber 
+- Ensure a great experience for our pilot users.
```

Comment:
How might you quantify "ensure a great experience for our pilot users"? Also, what does success look like w/r/t adoption numbers?

Reply:
Feel free to clarify this further and think about what you'd like it to be. In past quarterly plannings, I've seen objectives like "Get positive feedback from 5 pilot customers" specified.